libs/execution_context/ebpf_program.c

```diff
@@ -1654,10 +1655,14 @@ _ebpf_program_get_context(_Outptr_ void** context)
 typedef struct _ebpf_program_test_run_context
 {
     const ebpf_program_t* program;
+    ebpf_program_data_t* program_data;
     void* context;
     ebpf_program_test_run_options_t* options;
-    ebpf_signal_t* completion_event;
     uint8_t required_irql;
+    bool cancelled;
```

Comment:
```suggestion
    bool canceled;
```
Use one L for consistency with EBPF_CANCELED as in line 1693.

FYI per https://www.grammarbook.com/blog/spelling/cancelled-or-canceled/
cancelled is british, canceled is american (like colour vs color).  In this project we try to use american spellings.

Reply:
https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-erref/596a1078-e883-4972-9bbc-49e60bebca55

![image](https://user-images.githubusercontent.com/20480683/214720195-28b9b2f7-02fc-4455-9aab-68c13c403bef.png)

I am fine changing it, but Windows uses cancelled with two Ls.

Reply:
Yes Windows is internally inconsistent.  WinRT APIs corrected it to Canceled.